mingw-w64-bcg729/PKGBUILD

```
@@ -0,0 +1,46 @@
+# Maintainer: Wireshark Core Team <wireshark-dev@wireshark.org>
+
+_realname=bcg729
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.1.1
+pkgrel=1
+pkgdesc="Encoder and decoder of the ITU G.729 Annex A/B speech codec (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+url='https://www.linphone.org/technical-corner/bcg729/overview'
+license=('GPL2')
```

```suggestion
license=('spdx:GPL-3.0-or-later')
```

The license seems to be GPL-3.0+ https://github.com/BelledonneCommunications/bcg729/blob/1.1.1/LICENSE.txt